tests/tests.py

```diff
@@ -178,6 +180,44 @@ def test_my_permissions_by_issue(self):
         self.assertGreaterEqual(len(perms["permissions"]), 10)
 
 
+class MyPermissionsCloudTests(JiraTestCase):
```

Comment:
```suggestion
@allow_on_cloud
class MyPermissionsCloudTests(JiraTestCase):
```

This is imported from `tests.conftest`

The `skipTest` is fine here, if it doesn't work on server

Reply:
You're right, for readability let's add the ```allow_on_cloud``` mark. We still need the ```skip``` since the permission API in the Jira cloud has diverged from the Jira server.

Reply:
Ok some update on my comment, more than readability, ```allow_on_cloud``` **must** be present to ensure the test runs on the cloud :)

Reply:
yep exactly

https://github.com/pycontribs/jira/blob/009ec9fa50e482df153e695d67b11690ed0b5eea/.github/workflows/jira_cloud_ci.yml#L59